SimG4CMS/Calo/interface/CaloSD.h

```diff
@@ -49,23 +49,23 @@ class CaloSD : public SensitiveCaloDetector,
 
 public:    
   
-  CaloSD(G4String  aSDname, const DDCompactView & cpv,
+  CaloSD(std::string aSDname, const DDCompactView & cpv,
```

Comment:
Hi David, unfortunately I cannot because in the base G4 interface the string is not const....